### ISSUE TITLE
Creating Tables

### ISSUE THREAD

**Issue by SoftCircuits** [xoofx/markdig#477]
I've been successfully using MarkdownSharp for a while now but was hoping to get some more sophisticated support for things like tables.

So I tried the following code:

```cs
var pipeline = new MarkdownPipelineBuilder().UseAdvancedExtensions().Build();
ArticlePreview preview = new ArticlePreview(txtTitle.Text, DateTime.Today.ToString("d"), Markdig.Markdown.ToHtml(txtArticle.Text, pipeline), tglDeprecated.IsChecked ?? false);
```

And I tried to create a simple table:

```
|---|---|
| abc | def |
|---|---|
```

The the text above is not converted to a table.

Is this supported? What am I missing?

**Comment by rjgamble** [comment]
I think your markdown is missing a header definition. See below example:
```
| Header 1 | Header 2 |
| -------- | -------- |
| data 1 | data 2 |
```
Result in GitHub comment:

| Header 1 | Header 2 |
| -------- | -------- |
| data 1 | data 2 |

**Comment by nhamawi** [comment]
I spent several hours trying to get Markdig to convert a markdown table to HTML and I'm getting only rows of text wrapped in <p> and </p>. I'm using c#, Visual Studio 2019, and Markdig 0.24.0 from nuget. I'm creating a new MarkdownPipelineBuilder and adding .UsePipeTables(), but still no luck. The behavior is as if it's enforcing strict commonmark syntax which does not allow tables to be defined by using pipe symbols. I see no way to disable strict mode. However, I see documentation on this site for supporting pipe-based markup. Babelmark III also confirms that Markdig cannot support pipe-based markup which is in contrast to the documentation as I have pasted Markdig's own samples there and get the same (unexpected) result.
